DOCS-3024: Fix flutter code sample control motor (#3614)
viamrobotics/docs: docs/how-tos/control-motor.md

}
```

> This code creates a screen with a power widget to adjust the power and a button that, when pressed, calls on the `setPower()` method to set the power of the motor to 35% spinning the motor forwards, waits three seconds, and then stops the motor.

Author: ```suggestion
This code creates a screen with a power widget to adjust the power and a button that, when pressed, calls on the `setPower()` method to spin the motor forwards at 35% power, waits three seconds to let it spin, and then stops the motor.
```
